include/RAJA/policy/sycl/policy.hpp

```diff
@@ -39,7 +39,7 @@ struct uint3 {
   unsigned long x, y, z;
 };
 
-using sycl_dim_t = cl::sycl::range<1>;
+using sycl_dim_t = sycl::range<1>;
```

Comment:
Maybe keep the global scope `::sycl` for consistency?

```suggestion
using sycl_dim_t = ::sycl::range<1>;
```

Reply:
Good catch. Fixed.

Note that I didn't exhaustively search the code for these. I only changed the lines that generated compiler warnings.